School/utils/add_schools_to_db.go

```diff
@@ -37,12 +35,11 @@ func addSchoolsToDB() {
 	defer tx.Rollback(context.Background())
 
 	var schools []domain.School
-	resp, _ := http.Get("https://raw.githubusercontent.com/Hipo/university-domains-list/master/world_universities_and_domains.json")
-	schoolJSON, _ := ioutil.ReadAll(resp.Body)
+	jsonFile, _ := os.Open("school.json")
```

Comment:
I see you're reading from a json file now, but where do you generate this file? For this bug fix to be complete, the data from this URL must be loaded onto the json file first then you can read the data from it. As of now, this json file doesn't exist and the os.Open method will throw a file not found error. You only have to load the data from the URL once onto the json and include the json file in the utils directory in this PR.